CHANGELOG.md

```diff
@@ -79,6 +79,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - **Major** - Added: Multiworld and automatic item tracker support for Citra.
 - Added: New door types: `Access Open` and `Lightning Armor`. Access Open adds new doors to most 3-Tile high open transitions, which can then be shuffled. Lightning Armor doors can be opened with a Lightning Melee.
 - Added: Option to configure if heated rooms or lava deals constant instead of scaled damage.
+- Added: A guaranteed Baby Metroid hint displays in the textbox after collecting all DNA if you don't have the Baby.
```

Comment:
That's the future default but would still be nice if it mentions somehow that this is an configurable option.